src/SourceBuild/content/eng/tools/join-verticals.proj

```diff
@@ -0,0 +1,46 @@
+<Project>
+    <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+    <PropertyGroup>
+        <!-- Fake, to satisfy the SDK. -->
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    </PropertyGroup>
+
+    <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
```

Comment:
This project should use the Microsoft.Build.NoTargets SDK which is now allowed in the VMR:

```suggestion
<Project Sdk="Microsoft.Build.NoTargets">

    <PropertyGroup>
        <TargetFramework>$(NetCurrent)</TargetFramework>
    </PropertyGroup>
```

Reply:
I don't think we want this. I think it would be easier to use if the `join-verticals.proj` was self sufficient, and it built everything it needed to run so we can just call it and not worry about anything else

Reply:
That's possible with the above changes.

Reply:
The Microsoft.NET.Sdk should only be used in projects which compile. This project doesn't compile anything. It just references another project and has a target that invokes a task. Therefore NoTargets makes most sense here (it's also more performant).

Reply:
Ah I see thanks. When I was testing this it was asking me to restore the project before calling it, so I added that in the yaml. I looked at the Msbuild function in `tools.ps1`, couldn't find any parameter that would do that for me